### PR TITLE
Pn/derived keys

### PR DESCRIPTION
**Pull request by AeonSw4n**

### CODE REVIEW

**Review by maebeam — APPROVED** [review]

**Review by maebeam — APPROVED** [review]
Removing the unused `false` param would be great

**Review by maebeam — APPROVED** [review]

File: routes/transaction.go [hunk 1]
```diff
@@ -2038,3 +2038,114 @@ func (fes *APIServer) SendDiamonds(ww http.ResponseWriter, req *http.Request) {
...
+	MinFeeRateNanosPerKB uint64 `safeForLogging:"true"`
+}
+
+// AuthorizeDerivedKeyResponse ...
+type AuthorizeDerivedKeyResponse struct {
+	SpendAmountNanos  uint64
+	TotalInputNanos   uint64
+	ChangeAmountNanos uint64
+	FeeNanos          uint64
+	Transaction       *lib.MsgBitCloutTxn
```
Comment (AeonSw4n) [inline routes/transaction.go:2068]: DeSo

**Review by AeonSw4n — COMMENTED** [review]
in-class review

File: routes/admin_transaction.go [hunk 2]
```diff
@@ -305,3 +305,65 @@ func (fes *APIServer) SwapIdentity(ww http.ResponseWriter, req *http.Request) {
 		return
 	}
 }
+
+// SignTransactionWithDerivedKeyRequest ...
```
Comment (AeonSw4n) [inline routes/admin_transaction.go:309]: Comment this guy heavily

Comment (AeonSw4n) [reply routes/admin_transaction.go]: Done

**Pull request merged**